docs/Sway and wlroots support.md

```diff
@@ -29,6 +42,13 @@ To ensure that Flameshot is correctly positioned on multiple outputs (monitors)
 for_window [app_id="flameshot"] border pixel 0, floating enable, fullscreen disable, move absolute position 0 0
 ```
 
+and add the following on your River config:
+
+```
+riverctl rule-add -app-id "flameshot" float
+```
+
+Otherwise, flameshot will not take all of the screen and tiles its window instead like a normal application. Note however, that some clipboard stuff is broken so it might be good to save your screenshot as a file while having it copied to a clipboard in case if clipboard does some weird stuff like not pasting the overall screenshot.
```

Comment:
Did you build flameshot with WAYLAND_CLIPBOARD=true and still have clipboard issues?

Reply:
Oh, I just realized that part is old and no longer relevant. Let me fix that.